src/utils/types.ts

```diff
@@ -137,3 +137,6 @@ export function nullsToUndefined<T>(
   }
   return obj as any;
 }
+
+export const isNotUndefined = <S>(value: S | undefined): value is S =>
+  isNil(value) === false;
```

Comment:
Ugh can we avoid pulling this in? I've always found it a weird light and mildly inconsistent wrapper of `_.isNil()`.